core/scripts/common/vrf/jobs/jobs.go

```diff
@@ -27,15 +27,17 @@ vrf          [type=vrfv2
 estimate_gas [type=estimategaslimit
               to="%s"
               multiplier="%f"
-              data="$(vrf.output)"]
+              data="$(vrf.output)"
+              block=%s]
```

Comment:
Should this be

>              block="%s"]

?

Since block is a string

Reply:
Looks like it works with or without double quotes. Will add it for consistency